src/lib/ui/AccessCodeTable/AccessCodeTableContent.tsx

```diff
@@ -0,0 +1,61 @@
+import type { AccessCode } from 'seamapi'
+
+import {
+  TableBody,
+  TableCell,
+  TableHeader,
+  TableRow,
+  TableTitle,
+} from 'lib/ui/Table/index.js'
+
+import { AccessCodeKeyIcon } from 'lib/icons/AccessCodeKey.js'
+import { CodeDetails } from 'lib/ui/AccessCodeTable/CodeDetails.js'
+import { ContentHeader } from 'lib/ui/layout/ContentHeader.js'
+import { Caption } from 'lib/ui/typography/Caption.js'
+import { Title } from 'lib/ui/typography/Title.js'
+
+export interface AccessCodeTableContentProps {
+  accessCodes: AccessCode[]
+  onClickBack?: () => void
+}
+
+export function AccessCodeTableContent(
+  props: AccessCodeTableContentProps
+): JSX.Element {
+  const { accessCodes, onClickBack } = props
+
+  return (
+    <div className='seam-access-code-table'>
+      <ContentHeader onClickBack={onClickBack} />
+      <TableHeader>
+        <TableTitle>
+          {t.accessCodes} <Caption>({accessCodes.length})</Caption>
+        </TableTitle>
+      </TableHeader>
+      <TableBody>
+        {accessCodes.map((code) => (
+          <TableRow key={code.access_code_id}>
+            <TableCell className='seam-icon-cell'>
+              <div>
+                <AccessCodeKeyIcon />
+              </div>
+            </TableCell>
+            <TableCell className='seam-name-cell'>
+              <Title>{code.name}</Title>
+              <CodeDetails accessCode={code} />
+            </TableCell>
+            <TableCell className='seam-action-cell'>
+              {/* <IconButton>
```

Comment:
Is this attached to an issue?

Reply:
Yep, will be required when either of these are being implemented:

https://github.com/seamapi/react/issues/107
https://github.com/seamapi/react/issues/30

Reply:
Ok, can we remove the comments then? You can paste the code snippet in the issues if they are still relevant at that time.

Reply:
Damn, merged this a little early. I usually don't leave comments in case they're left dangling for eternity, but it's pretty safe to say this won't be missed -- those 2 features are required.

Do you want me to remove the comments in a separate PR?

Reply:
No worries. As long as they eventually disappear haha